exopy_hqc_legacy/instruments/drivers/visa/cryomagnetics_cs4.py

```diff
@@ -124,11 +128,14 @@ def sweep_to_field(self, value, rate=None):
         job = InstrJob(self.is_target_reached, wait, cancel=self.stop_sweep)
         return job
 
+    @secure_communication()
     def stop_sweep(self):
-        """Stop the field sweep at the current value.
+        """Stop the field sweep at the current value, and turn of the switch heater.
```

Comment:
Why this change in behavior ?

Reply:
It was specified in the job after job.cancel() was called (which in turn calls this method), it is more logical to integrate it directly to the method called as cancel()

Reply:
I disagree, you may stop a sweep but need to run a new one right after in which case automatically closing the switch heater would be a bad idea. Also if you wanted to go that way you need to be sure all driver are consistent.